MetaMask/metamask-extension
# Updating instances of "Metamask" to "MetaMask"

## Explanation

Currently, there are a few instances of "Metamask" in our translations, tests and comments.  This is an incorrect casing of "MetaMask"

This is a problem because it degrades our brand and causes confusion on the correct casing for our users and engineers - myself included!

In order to solve this problem, this pull request updates all instances of "Metamask" to "MetaMask" in content, tests and comments. It **DOES NOT** change any instance of "Metamask" in the code e.g. controllers, function names etc.

## More Information

This PR was spurred by @jspence425 [slack thread](https://consensys.slack.com/archives/CBHSSFEN5/p1654043719313349)

## Manual Testing Steps
- Search for instances of "Metamask" in VS code.
- Click the match case button [Aa]
- Check that all instances in translations, tests and comments have been updated to "MetaMask"

## Pre-Merge Checklist

- ~[x] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added~ NA
- ~[x] PR is linked to the appropriate GitHub issue~ NA
- ~[x] PR has been added to the appropriate release Milestone~ NA

### + If there are functional changes:

- ~[x] Manual testing complete & passed~ NA
- ~[x] "Extension QA Board" label has been applied~ NA
